src/org/loklak/objects/MessageEntry.java

```diff
@@ -57,7 +57,7 @@
     // left boundary must be space since the @ is itself a boundary
     public final static Pattern USER_PATTERN = Pattern.compile("(?:[ (]|^)(@..*?)(?:\\b|$)");
     // left boundary must be a space since the # is itself a boundary
-    final static Pattern HASHTAG_PATTERN = Pattern.compile("(?:[ (]|^)(#..*?)(?:\\b|$)");
+    final static Pattern HASHTAG_PATTERN = Pattern.compile("(?:[ (]|^)#(..*?)(?:\\b|$)");
```

Comment:
@sudheesh001 There isn't a bug in hashtags, the hashtags work fine, but Loklak Search team suggested that hashtags strings are more useful than hashtags with `#` symbol. https://github.com/loklak/loklak_server/issues/1457#issuecomment-322570756